sema/commons/rdf_ogm/term_builder.py

```
@@ -0,0 +1,73 @@
+    def __init__(
+        self,
+        template: str = None,
+        namespace: str = None,
+        prefix: str = None,
+        suffix: str = None,
+        graph: Graph = None
+    ):
+        self._namespace, self._prefix, self._suffix = None, None, None
+
+        if template:
+            assert not (namespace or prefix or suffix), "{template} is mutually exclusive with {namespace, prefix, suffix}"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)